src/app/dataset-view/additional-components/dataset-settings-component/tabs/scheduling/dataset-settings-scheduling-tab.component.html

```diff
@@ -1,167 +1,222 @@
+                                                >Next time: {{ nextTime }}</span
+                                            >
+                                        </ng-template>
+                                    </div>
+                                </mat-radio-button>
+                            </mat-radio-group>
+                            <div class="text-muted fs-12 border-top border-bottom px-4 py-4 mt-4">
+                                Cron expression accepted values
+                                <div class="fs-12">1. minutes: 0-59 * , -</div>
+                                <div class="fs-12">2. hours: 0-23 * , -</div>
+                                <div class="fs-12">3. day of month: 1-31 * , - ?</div>
+                                <div class="fs-12">4. months: (JAN-DEC or 1-12) * , -</div>
+                                <div class="fs-12">5. day of week: (SUN-SAT or 1-7) * , - ?</div>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="mb-4 mt-2 d-flex justify-content-end me-4">
+                        <button
+                            type="button"
+                            class="btn btn-success"
+                            [disabled]="pollingForm.invalid"
+                            (click)="savePollingTriggers()"
+                            data-test-id="save-config-options"
+                        >
+                            Save Trigger
+                        </button>
+                    </div>
+                </form>
+            </div>
+            <div class="position-relative border border-1 rounded-3 w-50 align-self-start px-4 pb-4">
+                <form [formGroup]="ingestConfigurationForm">
+                    <span class="position-absolute text-muted group-box-label text-uppercase">Configuration</span>
+
+                    <div class="d-flex align-items-center mt-4">
+                        <input
+                            class="checkbox-uncacheable me-3"
+                            type="checkbox"
+                            data-test-id="fetchUncacheable"
+                            formControlName="fetchUncacheable"
+                        />
+                        <span class="d-inline-block">Fetch uncacheable</span>
+                    </div>
+                    <div class="border-top mt-4"></div>
+                    <div class="mt-4 d-flex justify-content-end">
+                        <button
+                            type="button"
+                            class="btn btn-success"
+                            (click)="saveIngestConfiguration()"
+                            data-test-id="save-polling-configuration"
+                        >
+                            Save configuration
+                        </button>
+                    </div>
+                </form>
+            </div>
```

Comment:
I recommend to split this on multiple components. Definitely, trigger choices will be useful in other settings as well, let's say GC schedule in future Admin dashboard, or maybe regular Soft Compacting schedule, once we implement it.

It seems natural to split by main panels: triggers and configuration.

Reply:
Fixed